styles.scss

```diff
@@ -0,0 +1,299 @@
+@import 'variables';
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  background: $first-color;
+  font-family: coiny;
+}
+
+body {
+  .container {
```

Comment:
take everything from body outside of the body. This nesting is not needed and can cause some problems in the future. 

Everything is a child of body ;)